post-processing/post_processing_config.yaml

```diff
@@ -0,0 +1,31 @@
+# Plot title
+title: "Example Plot"
+
+# All columns relevant to plot
+# Minimum required columns: [figure_of_merit_value, figure_of_merit_unit,
+# something_to_plot_against]
+columns: ["flops_value", "flops_unit", "tasks"]
+
+# Optional row filters (specify an empty list [] if none are required)
+# Filtering may be conducted on columns that are not specified
+# in the list of columns above (so that the column is filtered by but also dropped)
+# Filter format: [column_name, operator, value]
+filters: [["tasks", ">", 1], ["cpus_per_task", "==", 2]]
+
+# Optional setting to display several plots in the same graph
+# Dataset format: [column_name, value]
+datasets: [["system", "default"]]
+
+# Axes must have a value specified with a column name
+# Units can either be specified with a column name
+# or a custom label (including null (i.e. None) if units are N/A)
+
+x_axis:

```

Comment:
More checks to be added to `read_config`: the value and units for both axis should be the same as the `columns` given above.

Reply:
Depending on what we need additional columns for, we could potentially remove the columns from config entirely and just internally compile a necessary columns list based on axis values, units, and datasets.